telegram/ext/_aioratelimiter.py

```diff
@@ -196,12 +196,11 @@ async def _run_request(
             self._get_group_limiter(group) if group and self._group_max_rate else null_context()
         )
 
-        async with group_context:  # skipcq: PTC-W0062
-            async with base_context:
-                # In case a retry_after was hit, we wait with processing the request
-                await self._retry_after_event.wait()
+        async with group_context, base_context:
```

Comment:
does this guarantee that `group_context` is entered before the `base_context`?

Reply:
yes, scroll down a bit on https://docs.python.org/3/reference/compound_stmts.html#with